backend/activity_log/models.py

```diff
@@ -82,8 +82,8 @@ def debounce_create(
         event_type,
         actor_user,
         effected_user,
-        object_1,
-        object_2=None,
+        object_1,       # RecruitProjectReview object
```

Comment:
These comments are misleading. object_1 is not always a RecruitProjectReview. Sometimes it's a topic review. Sometimes it will be a ReviewTrust instance... The idea is that we can log events to do with any kind of object, not just this one kind.